Looks like the test that is failing is because of this pbr addition:
```
_________________________ test_python_and_js_licenses __________________________
[gw0] linux -- Python 3.8.6 /var/lib/awx/venv/awx/bin/python3.8
Traceback (most recent call last):
  File "/awx_devel/awx/main/tests/functional/test_licenses.py", line 125, in test_python_and_js_licenses
    raise Exception('Included licenses not consistent with requirements:\n%s' % '\n'.join(errors))
Exception: Included licenses not consistent with requirements:
 license for requirement pbr is missing
 ```

we either need to skip pbr somehow or add the license it looks like :expressionless:

--- a/awx/main/tests/functional/test_licenses.py
+++ b/awx/main/tests/functional/test_licenses.py
@@ -48,6 +48,7 @@ def test_python_and_js_licenses():
 
     def read_api_requirements(path):
         ret = {}
+        skip_pbr_license_check = False
         for req_file in ['requirements.txt', 'requirements_git.txt']:
             fname = '%s/%s' % (path, req_file)
 
@@ -65,7 +66,11 @@ def test_python_and_js_licenses():
                         name = name[:-4]
                     if name == 'receptor':
                         name = 'receptorctl'
+                    if name == 'ansible-runner':
+                        skip_pbr_license_check = True
                 ret[name] = {'name': name, 'version': version}
+        if 'pbr' in ret and skip_pbr_license_check:
+            del ret['pbr']
         return ret
 
     def read_ui_requirements(path):

